File: day011/day011.cpp
#include <iostream>

using namespace std;

int main()
{
    int x;
    cout<<"Bir sayı giriniz: ";
    cin >> x;
    
    if (x % 2 == 0)
        printf("Girdiğiniz sayı çifttir");
    else
        printf("Girdiğiniz sayı tektir");

    return 0;
}

File: day011/day011.cs

public class Day011
{
	public static void main(String[] args) {
		int x;
		java.util.Scanner kb = new java.util.Scanner(System.in);
		
		System.out.print("Bir sayı giriniz: ");
		x = kb.nextInt();
		
        	if (x % 2 == 0)
            		System.out.println("Girdiğiniz sayı çifttir");
        	else
            		System.out.println("Girdiğiniz sayı tektir");
	}
}
